src/lib/components/types/log-message.ts

```diff
@@ -0,0 +1,5 @@
+export interface LogMessage {
+	timestamp: number;
+	level: string;
+	message: string;
+}
```

Comment:
I think a more natural way to handle this type is with the component props of the `LogEntry` component. I will push a change you can review.